website/versioned_docs/version-4.x.x/plugins/maven-plugin.md

```diff
@@ -73,13 +73,13 @@ Generate GraphQL client code based on the provided GraphQL schema and target que
 
     ```xml
 
-    
-      
-        
+
```

Comment:
i think with the migration xml got messed up -> see https://expediagroup.github.io/graphql-kotlin/docs/plugins/maven-plugin-usage#complete-configuration-example

Reply:
Something must have happened in IntelliJ git revert